support/Pipelines/Bbh/EccentricityControl.py

```diff
@@ -0,0 +1,89 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+import warnings
+
+import pandas as pd
+import yaml
+
+from spectre.Pipelines.EccentricityControl.EccentricityControlGL import (
```

Comment:
Delete `GL` from this name, and the `EccentricityControlGL.py` file that you added. This file is already merged and available, no need to duplicate it.

Reply:
they are actually different files - I made some changes to EccentricityControl in the GL version

Reply:
Oh, in that case please make the changes you need in the existing file.